CondTools/HLT/src/AlCaRecoTriggerBitsRcdUpdate.cc

```diff
@@ -28,12 +28,14 @@
 // Rcd for reading old one:
 #include "CondFormats/DataRecord/interface/AlCaRecoTriggerBitsRcd.h"
 
-class AlCaRecoTriggerBitsRcdUpdate : public edm::EDAnalyzer {
+class AlCaRecoTriggerBitsRcdUpdate : public edm::one::EDAnalyzer<edm::one::WatchRuns> {
```

Comment:
I don't understand why this needs to watch runs. Both the `beginRun` and `endRun` methods are empty, so there's no need for them

Reply:
I agree, there seems to be no real need for watch runs. It can be updated it in a future PR, if it is OK with you.

Reply:
see https://github.com/cms-sw/cmssw/pull/36187